product/carbon-home/conf/identity/profile-mapping.yaml

```diff
@@ -123,4 +123,31 @@ profiles:
    readonly: false
    dataType: "email"
    regex: "*"
-   defaultValue: "user@example.com"
\ No newline at end of file
+   defaultValue: "user@example.com"
+ -
+  profileName: "username-recovery"
+  isAdminProfile: true
+  claims:
+  -
+   claimURI: "http://wso2.org/claims/givenname"
+   displayName: "First Name"
+   required: true
+   readonly: false
+   regex: "*"
+   dataType: "text"
+  -
+   claimURI: "http://wso2.org/claims/lastname"
+   displayName: "Last Name"
+   required: true
+   readonly: false
+   regex: "*"
+   dataType: "text"
+  -
+   claimURI: "http://wso2.org/claims/email"
+   displayName: "Email"
+   required: true
+   readonly: false
+   regex: "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*
```

Comment:
The given regex pattern doesn't work. You can use the following.
`^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$`